src/index.html

```diff
@@ -5,8 +5,324 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__href">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        class="header__logo"
```

Comment:
```suggestion
          class="header__logo"
```
Consider fixing indentations according to the code style guide